docs/NativeModulesAndroid.md

```diff
@@ -89,21 +89,43 @@ ReadableArray -> Array
 ### Register the Module
 
 The last step within Java is to register the Module; this happens in the `createNativeModules` of your apps package. If a module is not registered it will not be available from JavaScript.
+In the package class it is important you override three methods: `createNativeModules`, `createJSModules` and `createViewManagers`.
```

Comment:
Grammatically, I guess it should be,

In the package class**,** it is important **that** you override **the following** three methods

or

In the package class*, you need to* override *the following* three methods